src/tests/topp/THIRDPARTY/third_party_tests.cmake

```diff
@@ -79,7 +79,7 @@ OPENMS_FINDBINARY(FIDOCHOOSEPARAMS_BINARY "FidoChooseParameters" "FidoChoosePara
 
 #------------------------------------------------------------------------------
 # Sirius
-OPENMS_FINDBINARY(SIRIUS_BINARY "sirius;sirius-console-64.exe" "Sirius")
```

Comment:
https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms_pr_ci/3440/compiler=msvc-15,os_label=win10_64/artifact/build/General_Testing/Testing/Temporary/LastConfigure_20191011-1349.log/*view*/

I think this makes it not find sirius on windows. Where do we have a sirius.jar? The executable is still needed but only to find the right directory.

Reply:
https://abibuilder.informatik.uni-tuebingen.de/jenkins/job/openms_pr_ci/compiler=appleclang-9.0.0,os_label=highsierra/ws/SEARCHENGINES/Sirius/

This is how the structure looks like

Reply:
And we only add the Sirius directory to the PATH environment. I can change it to Sirius/lib or we can restructure the  Sirius folder. Or we revert that line.

Reply:
See below, was changed to the .jar - so it should not look for the "binary" at all.

Reply:
The sirius.jar - should be for all at 
THIRDPARTY/All/Sirius/lib 
and the glpk - is at the THIRDPARTY/Windows/64bit/Sirius/lib 

see "lib_class_path" - SiriusAdapterAlgorithm

Reply:
Ok, now I get it. It will not be tested since the "binary" is not found. 

The problem here is that it depends on what kind of config you are using. 
If you are using the THIRDPARTY module - the jar will be called: sirius_openms_plugin_4.0.1.jar
Using the "normal" CLT: sirius_cli_4.0.1.jar

We could set the "sirius_openms_plugin-4.0.1.jar" as a default. 
-> THIRDPARTY/All/Sirius/lib/sirius_openms_plugin-4.0.1.jar

Reply:
Ok. Yes we could do that as a default but:
1) why is it different? Should we change our third-party module?
2) do we need to point to (i.e. put into the PATH) the lib folder then? Because I think the jar is in the lib folder right?

Reply:
1. That was a decision we made, when SIRIUS was integrated in the THIRDPARTY module. To reduce the size of SIRIUS for all the three paltforms, all libraries were placed in to "All".  The glpk are OS dependent, that's why they are in the specific OS dirs. To keep calling SIRIUS consistent, there is a bash-script ("binary") in each OS dir to call the jar file. 

2. I think everything should be in the path as long as the .jar file is set correctly. Please check the "lib_class_path" in SiriusAdapterAlgorithm. 
libpath + "/*" - should actually point to the lib dir, since the .jar file is located there as well. 

How are the binaries usually set in jenkins?